torchtext/datasets/text_classification.py

```diff
@@ -157,20 +209,34 @@ def AG_NEWS(*args, **kwargs):
     Arguments:
         root: Directory where the datasets are saved. Default: ".data"
         ngrams: a contiguous sequence of n items from s string text.
-            Default: 1
+            Default: 2
```

Comment:
I'm not a fan of returning bigrams by default because:
a) the default vocab will be much larger
b) it's more intuitive in my opinion to get lines of tokens instead of tokens + bigram tokens
c) we can expect some users to maybe want to combine this with ngram iterator or similar abstractions, which can lead to nasty bugs
d) It's strongly BC-breaking. I understand this comes from the unsupervised dataset, I'd suggest we switch it over to "ngrams 1" instead of this to "2" by default.